src/app/chat-page/chat-page.component.ts

```diff
@@ -186,6 +186,25 @@ export class ChatPageComponent implements OnInit, AfterViewInit {
           Object.assign(record, record[varname]);
           delete record[varname];
         },
+        isAdult: (record: any) => {
+          let age = parseInt(record.age)
+          if (age >=18) {
+            return true
+          }
+          else {
+            return false
+          }
+        },
+        inInclusion: (record: any) => {
```

Comment:
where is this function used?

Reply:
@akariv we are waiting for Ori's approval regarding the relevant question. I would like to have the function ready for use.

Reply:
but it just a duplicate of the above function.

Reply:
fixed.